include/mbgl/storage/default_file_source.hpp

```diff
@@ -134,7 +129,7 @@ class DefaultFileSource : public FileSource {
      * executed on the database thread; it is the responsibility of the SDK bindings
      * to re-execute a user-provided callback on the main thread.
      */
-    void deleteOfflineRegion(OfflineRegion&&, std::function<void(std::exception_ptr)>, bool pack = true);
+    void deleteOfflineRegion(OfflineRegion&&, std::function<void(std::exception_ptr)>);
 
     /*
```

Comment:
In what order are the APIs in this file listed? Currently, the order appears to be neither alphabetical nor logical...

if the order can be changed without too much hassle, let's do so in a separate pr

Reply:
I like the idea :+1: @tobrun @LukasPaczos wdyt?